style.css

```diff
@@ -66,4 +68,223 @@
     font-weight: 500;
     font-size: 16px;
   }
+
+  .start-btn:hover {
+    background-color: white;
+    color: #1a2236;
+    border: 1px solid #1a2236;
+    transition: 300ms;
+  }
```

Comment:
- [ ] Kindly make sure to align correctly this button in order to meet the provided Figma design. :wink: 

   - ### Your design
   ![image](https://user-images.githubusercontent.com/87197412/191794520-26227339-bad4-475d-9f9e-deec8344c004.png)


   - ### Figma design
    ![image](https://user-images.githubusercontent.com/87197412/191794641-8ddd261c-3b22-4e0b-afbb-bd0c4cbabebe.png)